### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by marcossalto**
Add UI test
SignInActivityTest
SignUpActivityTest
ProfileActivityTest

Unify User network models

Screenshot_1614804788
Screenshot_1614804813
Screenshot_1614804829

### CODE REVIEW

**Comment by agustinkoll-rootstrap** [comment]
I think you have to point the branch to develop

**Comment by marcossalto** [comment]
> I think you have to point the branch to develop

Yes! that's right
Thanks for your suggestion

**Review by agustinkoll-rootstrap — APPROVED** [review]

File: app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @Json(name = "push_token") val pushToken: String?,
+    @Json( name ="allow_password_change") val allowPasswordChange: Boolean? = false
+)
+
+@JsonClass(generateAdapter = true)
+data class Avatar(
+    @Json(name = "url") val url: String? = null,
+    @Json(name = "normal") val normal: Normal = Normal(null),
+    @Json(name = "small_thumb") val smallThumb: SmallThumb = SmallThumb(null)
+    )
```
Comment (ximenaperez) [inline app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt]: this parenthesis should be at the start of the line

File: app/src/androidTest/java/com/marcossalto/targetmvd/utils/BaseTests.kt [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    open fun scrollAndTypeText(id: Int, text: String) {
+        onView(ViewMatchers.withId(id)).perform(
+            ViewActions.scrollTo(),
+            ViewActions.click(),
+            ViewActions.clearText(),
+            ViewActions.typeText(text),
+            ViewActions.closeSoftKeyboard()
+        )
+    }
+
```
Comment (ximenaperez) [inline app/src/androidTest/java/com/marcossalto/targetmvd/utils/BaseTests.kt:104]: remove the extra line

**Review by ximenaperez — APPROVED** [review]

File: app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt (hunk 1 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt]: Thanks for your suggestion

File: app/src/androidTest/java/com/marcossalto/targetmvd/utils/BaseTests.kt (hunk 2 above)
Comment (marcossalto) [reply app/src/androidTest/java/com/marcossalto/targetmvd/utils/BaseTests.kt:104]: Thank you. Done

**Pull request merged**